src/IO/Observer/Tags.hpp

```diff
@@ -223,5 +236,23 @@ struct ReductionFileName : db::SimpleTag {
     return reduction_file_name;
   }
 };
+
+/// \brief The name of the HDF5 file on disk into which surface data is
```

Comment:
In the future, could you please keep commits like these smaller. E.g. add the tags in their own commit, and add the action in its own commit?